Improved: CommonExt - Move from hard-code menu location to parameterized (OFBIZ-12951)

PierreSmits opened the pull request:
ofbizsetup uses profile screens from partymgr.

modified web.xml
- added profileMenuLocation

PierreSmits committed:
Improved: CommonExt - Move from hard-code menu location to parameterized (OFBIZ-12951)

ofbizsetup uses profile screens from partymgr.

modified web.xml
- added profileMenuLocation

JacquesLeRoux commented:
Hi @PierreSmits ,

After applying and a restart, I still get :
ERROR MESSAGE

java.lang.RuntimeException: Error rendering included menu named [MainActionMenu] at location [component://commonext/widget/ofbizsetup/Menus.xml]: java.lang.IllegalArgumentException: Could not find menu with name [MainActionMenu] in location [component://commonext/widget/ofbizsetup/Menus.xml]

Obviously MainActionMenu is not profileMenuLocation.

PierreSmits committed:
OFBIZ-12951 Fix dependency on party

ofbizsetup is dependent on party elements

modified: commonext
- web.xml: added new context-param for createNewParty menu

modified: party
- web.xml: added new context-param for createNewParty menu
- PartyScreens.xml: changed location createNewParty menu reference

PierreSmits committed:
Improved: CommonExt - Move from hard-code menu location to parameterized (OFBIZ-12951)

modified: Menus.xml
- added MainActionMenu extending MainActionMenu in PartyMenus.xml
